skimage/feature/blob.py

```diff
@@ -185,6 +185,32 @@ def _prune_blobs(blobs_array, overlap, *, sigma_dim=1):
     return np.array([b for b in blobs_array if b[-1] > 0])
 
 
+def _format_exclude_border(img_ndim, exclude_border):
+    """Format an ``exclude_border`` argument as a tuple of ints for calling
+    ``peak_local_max``.
+    """
+    if isinstance(exclude_border, tuple):
+        if len(exclude_border) != img_ndim:
+            raise ValueError(
+                "`exclude_border` should have the same length as the "
+                "dimensionality of the image.")
+        for exclude in exclude_border:
+            if not isinstance(exclude, int):
+                raise ValueError(
+                    "exclude border, when expressed as a tuple, must only "
+                    "contain ints.")
+        return exclude_border
+    elif isinstance(exclude_border, int):
+        return (exclude_border,) * img_ndim + (0,)
+    elif exclude_border is True:
+        raise ValueError("exclude_border cannot be True")
+    elif exclude_border is False:
+        return (0,) * (img_ndim + 1)
+    else:
+        raise ValueError(
+            "Unsupported value ({}) for exclude_border".format(exclude_border))
```

Comment:
You don't need to fix it for this PR, but note that we are now targeting Py 3.6 and above, so you can replace this with:

```suggestion
            f"Unsupported value ({exclude_border}) for exclude_border"
        )
```